frontend/lib/l10n/app_de.arb

```diff
@@ -0,0 +1,375 @@
+        "format": "compact"
+      }
+    }
+  },
+  "addNote": "Notiz hinzufügen",
+  "enterNote": "Notiz eingeben",
+  "selectAtLeastOnePlant": "Wähle mindestens eine Pflanze aus",
+  "selectAtLeastOneEvent": "Wähle mindestens ein Ereignis aus",
+  "eventSuccessfullyUpdated": "Ereignis erfolgreich aktualisiert",
+  "editEvent": "Ereignis bearbeiten",
+  "eventSuccessfullyDeleted": "Ereignis erfolgreich gelöscht",
+  "noInfoAvailable": "Keine Informationen verfügbar",
+  "species": "Arten",
+  "plant": "Pflanze",
+  "scientificClassification": "Wissenschaftliche Klassifikation",
+  "family": "Familie",
+  "genus": "Gattung",
+  "synonyms": "Synonyme",
+  "care": "Pflege",
+  "light": "Licht",
+  "humidity": "Feuchtigkeit",
+  "maxTemp": "Maximale Temperatur",
+  "minTemp": "Minimale Temperatur",
+  "minPh": "Minimaler pH-Wert",
+  "maxPh": "Maximaler pH-Wert",
+  "info": "Informationen",
+  "addPhotos": "Fotos hinzufügen",
+  "addEvents": "Ereignisse hinzufügen",
+  "modifyPlant": "Pflanze bearbeiten",
+  "removePlant": "Pflanze entfernen",
+  "useBirthday": "Geburtsdatum verwenden",
+  "birthday": "Geburtsdatum",
+  "avatar": "Avatar",
+  "note": "Notiz",
+  "stats": "Statistiken",
+  "eventStats": "Ereignisstatistiken",
+  "age": "Alter",
+  "newBorn": "Neugeboren",
+  "nDays": "{count} {count, plural, =1{Tag} other{Tage}}",
+  "@nDays": {
+    "description": "Wie viele Tage eine Entität alt ist",
+    "placeholders": {
+      "count": {
+        "type": "num",
+        "format": "compact"
+      }
+    }
+  },
+  "nMonths": "{count} {count, plural, =1{Monat} other{Monate}}",
+  "@nMonths": {
+    "description": "Wie viele Monate eine Entität alt ist",
+    "placeholders": {
+      "count": {
+        "type": "num",
+        "format": "compact"
+      }
+    }
+  },
+  "nYears": "{count} {count, plural, =1{Jahr} other{Jahre}}",
+  "@nYears": {
+    "description": "Wie viele Jahre eine Entität alt ist",
+    "placeholders": {
+      "count": {
+        "type": "num",
+        "format": "compact"
+      }
+    }
+  },
+  "nMonthsAndDays": "{months} {months, plural, =1{Monat} other{Monate}}, {days} {days, plural, =1{Tag} other{Tage}}",
+  "@nMonthsAndDays": {
+    "description": "Wie viele Monate und Tage eine Entität alt ist",
+    "placeholders": {
+      "months": {
+        "type": "num",
+        "format": "compact"
+      },
+      "days": {
+        "type": "num",
+        "format": "compact"
+      }
+    }
+  },
+  "nYearsAndDays": "{years} {years, plural, =1{Jahr} other{Jahre}}, {days} {days, plural, =1{Tag} other{Tage}}",
+  "@nYearsAndDays": {
+    "description": "Wie viele Jahre und Tage eine Entität alt ist",
+    "placeholders": {
+      "years": {
+        "type": "num",
+        "format": "compact"
+      },
+      "days": {
+        "type": "num",
+        "format": "compact"
+      }
+    }
+  },
+  "nYearsAndMonths": "{years} {years, plural, =1{Jahr} other{Jahre}}, {months} {months, plural, =1{Monat} other{Monate}}",
```

Comment:
I think this require one more `}`, something like this probably
```
"nYearsAndMonths": "{years} {years, plural, =1{Jahr} other{Jahre}}, {months} {months, plural, =1{Monat} other{Monate}}",
  "@nYearsAndMonths": {
    "description": "Wie viele Jahre und Monate eine Entität hat",
    "placeholders": {
      "years": {
        "type": "num",
        "format": "compact"
      },
      "months": {
        "type": "num",
        "format": "compact"
      }
    }
  },
```

Reply:
You anticipated me 😁
Let's see if now the frontend tests pass and if so we can merge